### DISCUSSION TITLE
阳明心学与“后心学”

### DISCUSSION THREAD

**Discussion by giscus** [zhuangbiaowei/zhuangbiaowei.github.io#3] [Announcements]
# 阳明心学与“后心学”

最近读完了钱穆先生的《阳明学述要》，我留了一个短评：“不是书不好，但是我反复思量，确实觉得阳明心学有问题。”

http://zhuangbiaowei.github.io/confucianism/ethics/2022/08/07/yangming-s-mindfulness-and-post-mindfulness.html

**Comment by jarodtang** [comment]
👍。非常有启发。 尤其是“我最希望的，就是成为一名孔子的学生，或者孟子的学生。”

我正好看了看王阳明（完全谈不上精读），中国人法古的话语，实在是有点让人头大，但细看下去确实是有非常好的东西。那就是“致良知”。“格物致知”和“正心诚意”的内在信仰，可能被儒家的话语给遮蔽了。而遮蔽那部分，我认为可能是最有价值的。

我想借用黄仁宇和余英时的（并不见得是他们本意）观点，现在我们可以站在大历史和更博大的视角来看待，王阳明、孔子和孟子。从这点，我是喜欢余英时那本书的（论天人之际）。那么，从孔子、老子（我最喜欢的）、王阳明，我们透彻的看下去，其实是一个基本信仰问题，就是“心与道合一”。深渊里面，其实就是我们文化最底层的信仰，天道。

从这个点看，现代心理学可能还无法彻底回答这个问题（虽然，我认为embodied cognition非常有潜力）。倒是有一次，杨振宁感动了我，有学生问，你是否相信有上帝。杨回答，如果没有上帝，世界怎么会如此协调和美？但，我不信上帝是某个人的样子。其实，这就是深渊里的天道。

这是，我的理解。
